pages/overview/visualizations/cntrib_issue_assignment.py

```diff
@@ -0,0 +1,331 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.issue_assignee_query import issue_assignee_query as iaq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+
+PAGE = "overview"
+VIZ_ID = "cntrib_issue-assignment"
+
+gc_cntrib_issue_assignment = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Contributor Issue Assignment",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            """
+                            Visualizes the delta number of issue assignments for each contributor \n
+                            that meets the assignment criteria. This can help understand the workload'\n
+                            distribution between each contributor or maintainer over time.
+                            """
+                        ),
+                    ],
+                    id=f"popover-{PAGE}-{VIZ_ID}",
+                    target=f"popover-target-{PAGE}-{VIZ_ID}",
+                    placement="top",
+                    is_open=False,
+                ),
+                dcc.Loading(
+                    dcc.Graph(id=f"{PAGE}-{VIZ_ID}"),
+                ),
+                dbc.Form(
+                    [
+                        dbc.Row(
+                            [
+                                dbc.Label(
+                                    "Total Assignments Required:",
+                                    html_for=f"assignments-required-{PAGE}-{VIZ_ID}",
+                                    width={"size": "auto"},
```

Comment:
I'm interested in your rationale for the value of this option. I don't intuitively see why someone would want to scope out contributors by their historical assignment volume but I may be missing some insight.

My perspective would be that I'd rather capture all assignments than miss any, although this option does allow for that. Maybe the default could be '1' for that case, and the user can decide if they have a more specific use-case than that.

Reply:
I found from playing with the intervals that this was the sweet spot of actually being able to see the graph and it not getting too granular for most use cases. From talking to people about how they would use this graph it would be to understand their teams workload so filtering out the users that only have a few makes it where the graph and legends are not over saturated. Id like to currate to the request and usecases we see and people can edit from there/we can change it if we get other feedback.

Reply:
Okay that use-case makes sense, I can see what you mean.